### PR TITLE
Feature/confirmation emails for application process

### PR DESCRIPTION
**Pull request by RMaksymczuk**
Hi,

I prepared mentee and mentor application confirmation emails containing application details. I also made sure that they will be send after the application will be created. 

Additionaly, I've added a rake task to send all missing confirmation emails.

I also prepared spec fabricators for mentee and metor applications. I wrote four small tests and I think we should keep doing this with all new features (sic!) :)

Please, test it and let me know if You'll notice anything that need fix ;)

### CODE REVIEW

**Comment by anjali-sharma** [comment]
like the idea of writing specs... 👍

File: app/views/mentee_application_mailer/confirm_application.html.slim [hunk 1]
```diff
@@ -18,3 +18,98 @@ p
...
+    .paragraph
+      div
+        strong First name:
+        span #{@application.first_name}
+      div
+        strong Last name:
+        span #{@application.last_name}
+      div
+        strong Country:
+        span #{@application.country}
```
Comment (DMips) [inline app/views/mentee_application_mailer/confirm_application.html.slim]: I think we should display full name of the country (we're storing its code in db) `Carmen::Country.coded(@application.country).to_s` or `Carmen::Country.coded(@application.country).name`

Comment (RMaksymczuk) [inline app/views/mentee_application_mailer/confirm_application.html.slim]: you're right, this way is much better :+1:

**Comment by RMaksymczuk** [comment]
cool, thanks :)

I made this small change you mentiond and I'm going to merg it and push to production

**Pull request closed without merging**